src/components/Navbar.astro

```diff
@@ -81,12 +82,20 @@ const navLinks = [
       }
     </div>
     <div class="hidden lg:flex lg:items-center lg:flex-1 lg:justify-end">
-      <a
-        href="/signup"
-        class="text-sm font-semibold leading-6 px-2 py-1 rounded-md bg-indigo-600 hover:bg-indigo-500 text-white"
+      <Button
+        link="/signup"
+        className="leading-6 !bg-transparent !text-primary-900 !shadow-none  !px-2 !py-1 !w-fit rounded-md mr-3"
```

Comment:
why are you passing class names in button?
Make conditional button component with variants

Reply:
That's just an alias and also "class" was having type error compatibility with preact and html tags